### ISSUE TITLE
Allow to optionally ignore certain properties in the diff

### ISSUE THREAD

**Issue by kwin** [jsedding/osgi-ds-metatype-diff#13]
The maven-scr-plugin allowed to add some global properties to each and every component (with the parameter `properties`, https://felix.apache.org/documentation/subprojects/apache-felix-maven-scr-plugin/apache-felix-maven-scr-plugin-use.html). This was used in Apache Sling to always generate a `service.vendor` (https://github.com/apache/sling-parent/blob/0322917ff27cf3674a3a1cf9ed26dc59a2c5a9ce/pom.xml#L372)

Also in the past it always incorrectly generated a `service.pid` property (https://issues.apache.org/jira/browse/FELIX-5304). It would be nice to have a command line argument which allows to skip properties with certain names because some properties are not relevant or deliberately left out.

**Comment by jsedding** [comment]
I agree on the requirement but don't have the bandwidth to implement it at the moment.

If you can work on a PR that would be great! Maybe it would be possible to describe differences to ignore using XPath or JSONPath in a very generic way. I.e. not only properties. WDYT?
